test/testGraphical.m

```diff
@@ -38,9 +38,9 @@
     %standalone mode!
     status = execute_type_stage(status, ipp);
 
-    if ~status.closeall && ~isempty(status.plotStage.fig_handle)
+    try
```

Comment:
Why `try`-`catch` instead of those explicit checks?
This would make the `closeall` option moot (which is fine by me), but I would like to understand why this approach is superior.